sycl/source/detail/os_util.cpp

```diff
@@ -35,23 +40,23 @@ struct ModuleInfo {
   const char *Name;     // out
 };
 
-static int callback(struct dl_phdr_info *info, size_t size, void *data) {
-  unsigned char *Base = reinterpret_cast<unsigned char *>(info->dlpi_addr);
-  ModuleInfo *MI = (ModuleInfo *)data;
+static int callback(struct dl_phdr_info *Info, size_t Size, void *Data) {
+  unsigned char *Base = reinterpret_cast<unsigned char *>(Info->dlpi_addr);
+  ModuleInfo *MI = (ModuleInfo *)Data;
+  const unsigned char *TestAddr =
+      reinterpret_cast<const unsigned char *>(MI->VirtAddr);
```

Comment:
In all these 3 lines, the type is already expressed in the `reinterpret_cast`, so use `auto Base =` and so on.
And then it should fit on 1 line and be even clearer.

Reply:
Ok, fixed. Thank you.